tools/print-unicode-ranges.sh

```diff
@@ -17,124 +17,170 @@
-         { \
-           if (range_begin == range_prev) \
-           { \
-             print range_begin, range_prev, range_begin_name; \
-           } \
-           else \
-           { \
-             print range_begin, range_prev,  range_begin_name, range_prev_name; \
-           } \
-         } \
-         \
-         { \
-           if (is_in_range == 0) \
-           { \
-             is_in_range=1; \
-             range_begin=$1; \
-             range_prev=$1; \
-             range_begin_name=$2; \
-             range_prev_name=$2; \
-           } \
-           else \
-           { \
-             if (range_prev + 1 == $1) \
-             { \
-               range_prev=$1; \
-               range_prev_name=$2
-             } \
-             else \
-             { \
-               output_next_range(); \
-               range_begin=$1; \
-               range_prev=$1; \
-               range_begin_name=$2; \
-               range_prev_name=$2; \
-             } \
-           } \
-         } \
-         \
-         END \
-         { \
-           output_next_range(); \
-         }' \
-         | awk \
-          'BEGIN \
-           { \
-             FS = ";" \
-           } \
-           { \
-             range_string = sprintf ("LIT_UNICODE_RANGE_'$UNICODE_CHAR_CATEGORY_UPPER_CASE' (%s, %s)", $1, $2); \
-             range_string_length = length (range_string); \
-             \
-             range_begin_name=$3; \
-             range_end_name=$4; \
-             \
-             range_begin_name_length = length (range_begin_name); \
-             range_end_name_length = length (range_end_name); \
-             \
-             printf "%s", range_string; \
-             if (range_end_name_length == 0) \
-             { \
-               printf " /* %s */\n", range_begin_name; \
-             } \
-             else \
-             { \
-               if (range_begin_name_length > range_end_name_length) \
-               { \
-                 indent1 = 0; \
-                 indent2 = range_string_length + range_begin_name_length / 2;
-                 indent3 = range_string_length + (range_begin_name_length - range_end_name_length) / 2; \
-               } \
-               else \
-               { \
-                 indent1 = (range_end_name_length - range_begin_name_length) / 2; \
-                 indent2 = range_string_length + range_end_name_length / 2;
-                 indent3 = range_string_length; \
-               } \
-               indent3 = indent3 + 3; \
-               fmt1 = sprintf (" /* %%%ds%%s\n", indent1); \
-               fmt2 = sprintf (" %%%ds<--->\n", indent2); \
-               fmt3 = sprintf (" %%%ds%%s */\n", indent3); \
-               \
-               printf fmt1, "", $3; \
-               printf fmt2, ""; \
-               printf fmt3, "", $4; \
-             } \
-             \
-             printf "\n"; \
-           }'
+awk -v desired_category="$CATEGORY" \
+'BEGIN \
+ { \
```

Comment:
I think this should be aligned to one more space, so the alignment is divisible by 2.